ApeWorX/ape
# feat: add sepolia network under ethereum ecosystem [APE-694]

### What I did

* add the sepolia network to `ape_ethereum`
* add the sepolia network to `ape_geth`

### How I did it

By adding references to it where it seemed suitable

### How to verify it

```sh
ape console --network etherum:sepolia:[rpc-url]
ape console --network etherum:sepolia:geth 
```
Haven't tried the second one, see the [geth PR about sepolia support](https://github.com/ethereum/go-ethereum/pull/23730)

### Checklist



- [x] New test cases have been added (more specifically : test data to an existing case)
- [x] Documentation has been updated (the new network is shown in the `ape networks list` command output, if that fells under documentation)
